bots/hermes/Makefile

```diff
@@ -0,0 +1,61 @@
+SHELL := /bin/bash
+.DEFAULT_GOAL=help
+
+VENVPATH=venv
+PYTHON=$(VENVPATH)/bin/python3
+
+export GOOGLE_APPLICATION_CREDENTIALS=.secrets/service-account.json
+
+##venv-activate: Activate the virtual environment
+##
+venv-activate:
```

Comment:
We can't rely on the activation of the virtual env to run our application unless we carefully deactivate after the application is run which is not the case in this Makefile